app/src/main/AndroidManifest.xml

```diff
@@ -50,6 +50,11 @@
             android:name=".activities.HostActivity"
             android:screenOrientation="portrait">
         </activity>
+        <activity
+            android:name=".activities.HostNewSpotActivity"
+            android:label="@string/title_activity_host_new_spot"
+            android:theme="@style/AppTheme">
```

Comment:
Pretty sure we don't need to give it a theme or a label. The label, from what I understand, is what gives it a weird name when looking at the active apps menu. Doesn't appear to be doing it here however so we don't have to worry about it I guess.